### ISSUE TITLE
Add SMTP configuration

### ISSUE THREAD

**Issue by diegorojas** [groucho75/alo-easymail#6]
What do you think about the idea of make a SMTP configuration tool for this plugin?

We Can make an option to be global or just for ALO easymail. It will be interesting for the plugin be more flexible, giving options for deliver emails from others remote servers. 

Made that, we can make happen with Mandrill from MailChimp and it will be awesome!

**Comment by groucho75** [comment]
A good suggestion. Some time ago I started investigating on it, I like too to add a SMTP setup in future releases.

**Comment by diegorojas** [comment]
Hey Man,

Nice! 

If you don't mind, me and my team at Brasa wants to work on that. This plugin is very usefull for us right now, we can help you to make it better.

What do you think?

**Comment by groucho75** [comment]
I'm very happy about it, you and your team are welcome. Please work on it.
BTW I'm very sorry about the plugin code that is a mess... In the past I never had the time to clear and reorganize it.
In future I'd like to make a refactor for v.3. But for now I have enough time only for little updates, so every help is welcome.

**Comment by jeffmcneill** [comment]
SMTP configuration usually puts account and password in the database in cleartext. This should be avoided. Suggest recommending the use of Postman SMTP plugin instead, as that can support OAuth2. Also, if there is some way of having the Bounce setting use that (or just the standard PHP/Wordpress mail calls that Postman SMTP takes over).

**Comment by jeffmcneill** [comment]
Hi Alo, not sure if you use Postman SMTP but it works well and keeps everything encrypted (and works with many email providers). If you added that to the FAQ it could help people use a product that is more secure than most. 

The only problem is that for your bounces you do have SMTP settings that are separate (which are cleartext in the database).

**Comment by groucho75** [comment]
I agree, I've updated the faq about smtp with Postman info: http://www.eventualo.net/blog/wp-alo-easymail-newsletter-faq/#faq-15
About bounces, I think Postman can set smtp only for outgoing emails. From its readme: _[Compatibile With....] "every other plugin that uses the WordPress API wp_mail to send mail"_. EasyMail bounce script makes a remote imap connection to a mailbox to read messages. We should investigate about integration with Postman.
Anyway, EasyMail does not store the mailbox password in database: the bounce routine is manual, not automatic, so the password is not stored but has to be typed each time.

**Comment by jeffmcneill** [comment]
Ah, thanks for that I had misunderstood. I'm unable to get the bouncing to work with Gmail/Google Apps, likely due to the SSL requirement?

It would be nice if it could use postman, but likely that only supports delivery (SMTP) not reading (IMAP)? Here are some google docs in any case. If it were possible to use the Postman's Certs that would be great, if not, duplicate the functionality and I can create a new OAuth2 cert. 

https://developers.google.com/gmail/oauth_overview
